packages/ember-cli-code-coverage/index.js

```diff
@@ -72,7 +88,13 @@ module.exports = {
     }
 
     const IstanbulPlugin = require.resolve('babel-plugin-istanbul');
-    return [[IstanbulPlugin, { cwd, include: '**/*', exclude }]];
+    return [
+      // String lookup is needed to workaround https://github.com/embroider-build/embroider/issues/1525
```

Comment:
As it turns out... We're going to require string lookup eventually, and forbid passing JS references.

JS references are not parallizable, but string references are! (it's a babel thing :shrug: )

Reply:
one thing you could try is using a `resolve`d string, so you get the absolute path of the plugin for "this instance of ember-cli-code-coverage".

```js
path.resolve('the-string-you-have'); // returns absolute string, so no matter who is requiring, their resolve isn't local to who is doing the requiring 
```